src/component/header/index.tsx

```diff
@@ -45,6 +45,13 @@ export default function Header(props: HeaderProps) {
         ref: dropdownRootRef,
       }}
       button={{
+        props: {
```

Comment:
Discovered an unrelated bug with navigation while testing.

Reply:
It's a bummer that I had not reviewed your PR earlier, it costed me hours to figure out on the PR. The problem is that button node is actually <button> tag. It is not recommended to wrap <a> around <button> due to the unexpected onClick behavior. If we do this, we don't need the wrap(node) below